omod/src/main/webapp/fragments/header.gsp

```diff
@@ -72,75 +72,102 @@
             });
 
             <% if (enableUserAccountExt) { %>
-            jq('.identifier').hover(
-                function(){
-                    jq('.appui-toggle').show();
-                    jq('.appui-icon-caret-down').hide();
-                },
-                 function(){
-                    jq('.appui-toggle').hide();
-                    jq('.appui-icon-caret-down').show();
-                 }
-            );
+            var event = ('ontouchstart' in window) ? 'click' : 'mouseenter mouseleave';
+
+            jq('.identifier').on(event,function(){
+                    jq('.appui-toggle').toggle();
+                    // jq(this).toggleClass("appui-toggle");
+                    jq('.appui-icon-caret-down').toggle();
+            });
+                
+            // jq('.identifier').hover(
+            //     function(){
+            //         jq('.appui-toggle').show();
+            //         jq('.appui-icon-caret-down').hide();
+            //     },
+            //      function(){
+            //         jq('.appui-toggle').hide();
+            //         jq('.appui-icon-caret-down').show();
+            //      }
+            // );
+
+
+            // if(!!('ontouchstart' in window)){
+            //     jq('.identifier').unbind('click mouseenter mouseleave hover'); 
+            //     jq('.identifier').on('click',function(){
+            //         jq('.appui-toggle').show();
+            //         jq('.appui-icon-caret-down').hide();
+            //     },
+            //      function(){
+            //         jq('.appui-toggle').hide();
+            //         jq('.appui-icon-caret-down').show();
+            //      });
+            // }
+  
             jq('.identifier').css('cursor', 'pointer');
             <% } %>
         <% } %>
     });
 
 </script>
-
 <header>
-    <div class="logo">
-        <a href="${ logoLinkUrl }">
-            <img src="${ logoIconUrl }"/>
-        </a>
-    </div>
     <% if (context.authenticated) { %>
-    <ul class="user-options">
-        <li class="identifier">
-            <i class="icon-user small"></i>
-            ${context.authenticatedUser.username ?: context.authenticatedUser.systemId}
-            <% if (enableUserAccountExt) { %>
-            <i class="icon-caret-down appui-icon-caret-down link"></i><i class="icon-caret-up link appui-toggle" style="display: none;"></i>
-                <ul id="user-account-menu" class="appui-toggle">
-                    <% userAccountMenuItems.each{ menuItem  -> %>
-                    <li>
-                        <a id="" href="/${ contextPath }/${ menuItem.url }">
-                            ${ ui.message(menuItem.label) }
-                        </a>
-                    </li>
+        <nav class="navbar navbar-expand-lg bg-dark navbar-dark" style="line-height: 1em;border-radius: 5px;color: #ccc;">
```

Comment:
Seems like the style is missing the lint. Why you prefer inline styles rather than components?

Reply:
@Akayeshmantha FYA

Reply:
Oh sorry missed this nothing inline sorry for it.This was code for do the testing and forgot to include it scss.